wiki/Glossary/Nintendo_DS_Games/en.md

```diff
@@ -0,0 +1,21 @@
+---
+tags:
+  - nintendo games
+  - ds games
+---
+
+# Nintendo DS Games
+
+The **Nintendo DS Games** or *DS Games* for short, were a set of rhythm games developed by [iNiS](https://en.wikipedia.org/wiki/INiS) for the [Nintendo DS](https://en.wikipedia.org/wiki/Nintendo_DS) in the early 2000s.
+
+The gameplay of these games inspired the creation of the [osu!standard](/wiki/Game_Modes/osu!) gamemode, and their level-design influenced the early years of osu!standard mapping heavily.
```

Comment:
```suggestion
In regards to osu! in the osu! wiki, *the* **Nintendo DS Games** (or *DS Games* for short,) were a set of original rhythm games developed by [iNiS](https://en.wikipedia.org/wiki/INiS) for the [Nintendo DS](https://en.wikipedia.org/wiki/Nintendo_DS) in the early 2000s. The gameplay of these games inspired the creation of the [osu!standard](/wiki/Game_Modes/osu!) [game mode](/wiki/Game_Modes). The DS games' level design influenced the early years of osu!standard mapping heavily.
```
Italicized "the" to clarify that, when talked about in regards to osu! in the osu! wiki specifcally, "DS games" refers specifically to the iNiS rhythm games.

Reply:
clarification is unnecessary since it is implied by being on the osu!wiki that whatever is written is related to osu! in some way or another but i'll try to make it shorter/sweeter and keep the intention the same